api/docs/v1.52.yaml

```diff
@@ -10855,7 +10815,39 @@ paths:
           required: true
           description: "Volume configuration"
           schema:
-            $ref: "#/definitions/VolumeCreateOptions"
+            type: "object"
```

Comment:
Hm..  probably instead we should just rename the definition; I tried to keep all the structs in `definitions` (not sure if 100% correct, but keeping them all together to make the endpoints somewhat less hard to read, AND, in some cases because the same structs are usable for both Create, Inspect, List)


We also still need (see my other comment) an actual type definition in the API for the request.